README.md

```diff
@@ -6,7 +6,7 @@
     Simple extensible ASS subtitle editor for Linux
 </p>
 
-## Features
+## Features [![Travis Build Status](https://travis-ci.org/rr-/bubblesub.svg?branch=master)](https://travis-ci.org/rr-/bubblesub)
```

Comment:
It's a weird place to put this, let's put it centered above this heading

Reply:
I prefer where it was, but done.